ui/src/pages/ResultsPage.tsx

```diff
@@ -2841,6 +2843,62 @@ const SeverityFilterField = (props: SeverityFilterFieldProps) => {
 	);
 };
 
+interface SecretValidityFilterFieldProps {
+	value?: string | string[];
+	autoFocus?: boolean;
+	onChange: (
+		event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+		field: string,
+		value: string
+	) => void;
+}
+
+const SecretValidityFilterField = (props: SecretValidityFilterFieldProps) => {
+	const { i18n } = useLingui();
+	const { classes } = useStyles();
+	const { value = "", autoFocus = false, onChange } = props;
+
+	const menuItems = Object.values(SecretValidity).map((value) => {
```

Comment:
(nit) `value` in the lambda shadows the `value` var from the surrounding function which can be confusing.

Reply:
Good catch